src/resolvers/item/resolver.ts

```diff
@@ -1,39 +1,57 @@
 import { Repository } from 'typeorm';
 import { InjectRepository } from 'typeorm-typedi-extensions';
-import { Resolver, Query, Arg, Int, Mutation } from 'type-graphql';
+import { Resolver, Query, Arg, Int, Mutation, Ctx, FieldResolver, Root, Authorized } from 'type-graphql';
 
 import Item from '@/entities/Item';
+import User from '@/entities/User';
 
+import { Context } from '../types';
 import { AddItemInput, UpdateItemInput } from './types';
 
 @Resolver(Item)
 export class ItemResolver {
   constructor(
     @InjectRepository(Item) private readonly itemsRepository: Repository<Item>,
+    @InjectRepository(User) private readonly userRepository: Repository<User>,
   ) {}
 
-  @Query(() => String)
-  public hello() {
-    return '1233';
-  }
-
   @Query(() => Item, { nullable: true })
   public async item(@Arg('itemId', () => Int) itemId: number) {
     return this.itemsRepository.findOne({ id: itemId });
   }
 
   @Query(() => [Item])
   public async items() {
-    return this.itemsRepository.find();
+    return this.itemsRepository.find({
+      order: {
+        id: 'DESC',
+      },
+    });
   }
 
+  @Authorized()
   @Mutation(() => Item)
-  public async addItem(@Arg('item') itemInput: AddItemInput) {
-    const item = this.itemsRepository.create({ ...itemInput });
+  public async addItem(@Arg('item') itemInput: AddItemInput, @Ctx() ctx: Context) {
+    let user;
+    if (ctx.me) {
+      user = await this.userRepository.findOne({ uuid: ctx.me.uuid });
+    } else {
+      const users = await this.userRepository.find({
```

Comment:
Should throw an error if the user is not found.

Reply:
It is a redundancy code, because I already use `@Authroized()` decorator .
I forgot to remove it.